lms/djangoapps/bulk_email/tests/test_course_optout.py

```diff
@@ -87,6 +90,30 @@ def test_optout_course(self):
         self.assertEqual(len(mail.outbox), 1)
         self.assertEqual(mail.outbox[0].to[0], self.instructor.email)
 
+    def test_optout_using_unsubscribe_link_in_email(self):
+        """
+        Make sure email (with Unicode characters) send to all goes there.
```

Comment:
nit: update doc string `Make sure email is not sent if learner unsubscribed from course email.`